### PR TITLE
deployment/alerts: add RemoteWriteDroppingData to vmalert rules

### PR DESCRIPTION
**Pull request by hagen1778**
### Describe Your Changes

Please provide a brief description of the changes you made. Be as specific as possible to help others understand the purpose and impact of your modifications.

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by AndrewChubatiuk — APPROVED** [review]

File: deployment/docker/rules/alerts-vmalert.yml [hunk 1]
```diff
@@ -83,3 +83,14 @@ groups:
           summary: "vmalert instance {{ $labels.instance }} is failing to send notifications to Alertmanager"
           description: "vmalert instance {{ $labels.instance }} is failing to send alert notifications to \"{{ $labels.addr }}\".
             Check vmalert's logs for detailed error message."
+
+      - alert: RemoteWriteDroppingData
```
Comment (Haleygo) [inline deployment/docker/rules/alerts-vmalert.yml:87]: It looks like a valid rule, but duplicates with rule `RemoteWriteErrors`(only differing in`for` and I don't know why)) since their metrics change simultaneously, so user will receive two alerts with the same messages. Is this expected?

Comment (hagen1778) [reply deployment/docker/rules/alerts-vmalert.yml:77]: Yes, the new alerting rule has clear title explaining consequences of what's happening.
RemoteWrite errors could be perceived as something temporary, while dropped data is immediately perceived as permanent effect. 
tl;dr; I expect this rule to be treated a way more seriously than RW errors.
